packages/react-components/src/components/styles/InputBoxComponent.style.ts

```diff
@@ -102,7 +102,8 @@ export const inlineButtonsContainerStyle: IStyle = {
   right: '0.3rem',
   top: '0',
   bottom: '0',
-  gap: '0.25rem'
+  gap: '0.25rem',
+  alignItems: 'center'
```

Comment:
This is a shared css style across sendbox and editbox (when you edit a message), could you also double check the edit box button style just in case?